inttest/airgap/airgap_test.go

```diff
@@ -36,6 +36,14 @@ type AirgapSuite struct {
 	common.FootlooseSuite
 }
 
+const network = "airgap"
+
+// SetupSuite creates the required network before starting footloose.
+func (s *AirgapSuite) SetupSuite() {
+	s.Require().NoError(s.CreateNetwork(network))
```

Comment:
We probably also want the appropriate MaybeDestroyNetwork call...

Reply:
D'oh! It's fixed now.